src/python/tritonfrontend/_api/logging.py

```diff
@@ -0,0 +1,5 @@
+from tritonfrontend._c.tritonfrontend_bindings import setLoggingVerbose
+
+
+def enable_logging(level):
```

Comment:
Shouldn't be public API

Reply:
Removed. Logging will be added in a follow up PR.